### ISSUE TITLE
Extension error when launching with VSCode F5

### ISSUE THREAD

**Issue by WilliamVenner** [rust-analyzer/rust-analyzer#9174]
I cloned the repo from GitHub, went into `editors/code` and did `npm install`, then ran "Run Extension (Debug Build)" and I get the following errors:

```
[matklad.rust-analyzer]: configuration.properties property '$generated-start' must be an object
[matklad.rust-analyzer]: configuration.properties property '$generated-end' must be an object
```

**Comment by lnicola** [comment]
See https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/debugging.md. To install the extension you need to run `catgo xtask install --client code` (or something like that). Run it without that argument to install both the extension and the language server.

**Comment by lnicola** [comment]
That might be a bug. I think you can manually edit `package.json` and remove those two lines.

**Comment by WilliamVenner** [comment]
Hm, removing them now just shows "Indexing 0/11 (core)" forever.

**Comment by Veykril** [comment]
Fixed in #9282
